recipe/configure.win

```diff
@@ -2,7 +2,7 @@
 
 set -euxo pipefail
 
-echo "PKG_CPPFLAGS=-DNDEBUG -I\"${LIBRARY_PREFIX}/include\" -I\"${PREFIX}/include\" -DARROW_R_WITH_ARROW -DARROW_R_WITH_S3" > src/Makevars.win
+echo "PKG_CPPFLAGS=-DNDEBUG -I\"${LIBRARY_PREFIX}/include\" -I\"${PREFIX}/include\" -DARROW_R_WITH_ARROW -DARROW_R_WITH_PARQUET -DARROW_R_WITH_DATASET -DARROW_R_WITH_S3" > src/Makevars.win
```

Comment:
Can you add a test command in the `meta.yml` under `test: commands:` that checks the flags are definitely building the feature?

We would need these flags for unix builds, too?

Reply:
> We would need these flags for unix builds, too?

I don't think so; the issue here is that the conda recipe patches/overwrites Makevars.win, but I don't think it does for mac/linux. Also, these flags are determined dynamically in the configure script at build time, so if the arrow-cpp library has these features enabled (which it does), the flags will be added.